app/web/src/store/module.store.ts

```diff
@@ -101,12 +100,27 @@ export type RemoteModuleDetails = RemoteModuleSummary & {
   };
 };
 
+// Gather the current changeset ID, since our components don't appear to be
+// reacting to the visibility changes that happen on the setup code
+// TODO: Generalize this and make the api client pass these arguments implicitly
+async function getVisibilityParams(forceChangeSetId?: ChangeSetId) {
```

Comment:
nit: doesn't need to be `async`?

Reply:
True. This is leftover from a version that would await for some requests and didn't end up getting implemented. Fixed it